test/built-ins/Proxy/set/trap-is-undefined-receiver.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2016 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.9
```

Comment:
Same here for esid: sec-proxy-object-internal-methods-and-internal-slots-set-p-v-receiver